Description: Fix FTBFS with gcc-4.7
 Fix missing inclusion of <unistd.h> in src/filesys.cpp.
Author: "P. J. McDermott" <pjm@nac.net>
Bug-Debian: http://bugs.debian.org/667280
Last-Update: 2012-04-19

--- a/src/filesys.cpp
+++ b/src/filesys.cpp
@@ -166,6 +166,7 @@
 
 #else // POSIX
 
+#include <unistd.h>
 #include <sys/types.h>
 #include <dirent.h>
 #include <errno.h>
